templates/ec2-bootstrap-trigger.yaml

```diff
@@ -0,0 +1,113 @@
+AWSTemplateFormatVersion: '2010-09-09'
+
+Description: >-
+  An EC2 for running the bootstrap-trigger script and triggering the
+  JSON to Parquet workflow on a recurring basis
+
+
+Parameters:
+
+  SsmParameterName:
+    Type: String
+    Description: Name of the SSM parameter containing the Synapse auth token
+    Default: synapse-bridgedownstream-auth
+
+  PrivateKey:
+    Type: String
+    Description: Name of the private key to access this instance via ssh
+
+  CrontabURI:
+    Type: String
+    Description: S3 URI of crontab file to install on this EC2 instance
+
+Resources:
+
+  EC2Instance:
+    Type: AWS::EC2::Instance
+    Properties:
+      InstanceType: t3.medium
+      ImageId: ami-03ededff12e34e59e
+      IamInstanceProfile: !Ref IAMInstanceProfile
+      KeyName: !Ref PrivateKey
+      SecurityGroupIds:
+        - !Ref SecurityGroup
+      UserData:
+        Fn::Base64:
+          !Sub >-
+          #! /bin/sh
+          yum update -y
+          amazon-linux-extras install docker
+          service docker start
+          usermod -a -G docker ec2-user
+          chkconfig docker on
+          aws ecr --region us-east-1 get-login-password | docker login --username AWS --password-stdin 634761300905.dkr.ecr.us-east-1.amazonaws.com/bootstrap_trigger
+          docker pull 634761300905.dkr.ecr.us-east-1.amazonaws.com/bootstrap_trigger
+          aws s3 cp ${CrontabURI} /tmp/crontab
+          crontab -u ec2-user /tmp/crontab
+
+  SecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: !Sub "The security group for ${AWS::StackName}"
+      SecurityGroupIngress:
```

Comment:
Why this security group?  also there probably needs to be tighter control on this instance.  We should probably put it behind our VPN.

Reply:
Adding VpcId - something like this: https://github.com/nlpsandbox/nlpsandbox-infra/blob/main/sceptre/nlpsandbox/templates/nlpsandbox.yaml#L188..

Do we have a VPC set up in this?

Reply:
I made another commit to add the security group to the VPC

Reply:
@thomasyu888 Does the EC2 need to be in a VPC because of the ingress rules? I should be able to remove this rule so that it's not possible to connect to the instance. Everything is taken care of by the EC2 user data. The ssh access is just for debugging purposes.